edumfa/lib/policy.py

```diff
@@ -789,11 +795,8 @@ def _policy_matches_request_environ_condition(policy, key, comparator, value, re
                     raise PolicyError(
                         "Invalid comparison in the HTTP environment conditions of policy {!r}".format(policy['name']))
             else:
-                log.warning("Unknown HTTP environment key referenced in condition of policy "
```

Comment:
Why are these log messages removed?

Reply:
The warning was moved to the function `_react_to_unset`.
The warning is only logged when the new functionality is not configured or set to `exception`